src/admin/resolvers/queries/Collection.integration.ts

```diff
@@ -223,16 +223,51 @@ describe('admin queries: Collection', () => {
 
       // create label and collection-label association
       fakeLabel = await createLabelHelper(db, 'region-east-africa', 'fakeUser');
+      fakeLabel2 = await createLabelHelper(
+        db,
+        'region-south-africa',
+        'fakeUser'
+      );
       const collectionLabelInputData: CreateCollectionLabelInput = {
         collectionId: fakeCollection.id,
         labelId: fakeLabel.id,
         createdAt: new Date(),
         createdBy: 'fakeUser',
       };
+      const collectionLabelInputData2: CreateCollectionLabelInput = {
+        collectionId: fakeCollection.id,
+        labelId: fakeLabel2.id,
+        createdAt: new Date(),
+        createdBy: 'fakeUser',
+      };
       await createCollectionLabelHelper(db, collectionLabelInputData);
+      await createCollectionLabelHelper(db, collectionLabelInputData2);
+    });
+
+    it('should search by all labelExternalIds in a single collection', async () => {
+      const { data } = await server.executeOperation({
+        query: SEARCH_COLLECTIONS,
+        variables: {
+          filters: {
+            labelExternalIds: [fakeLabel.externalId, fakeLabel2.externalId],
+          },
+        },
+      });
+
+      const collections = data?.searchCollections?.collections;
+
+      expect(collections.length).to.equal(1);
+      expect(collections[0].title).to.equal('the dude abides man');
+      expect(collections[0].labels.length).to.equal(2);
+      expect(collections[0].labels[0].externalId).to.equal(
+        fakeLabel.externalId
+      );
+      expect(collections[0].labels[1].externalId).to.equal(
+        fakeLabel2.externalId
+      );
     });
 
-    it('should search by labelExternalIds', async () => {
+    it('should search by one labelExternalId in collection(s) with more than 1 label', async () => {
```

Comment:
i think it would be worth testing two other use cases:

1. searching by multiple labels, making sure the results return a collection that only has one of the searched labels (essentially ensuring the OR logic is working as expected)
2. making sure no results are returned when no collections have the searched label

Reply:
the first point is already tested --- a collection has 2 labels but searched by one of the labels only